installer/webapp/src/main/webResources/META-INF/context.xml

```diff
@@ -1,8 +1,51 @@
 <Context> <!-- useHttpOnly="false" -->
+
+   <!--
+     # The 'home' property specifies the location of Vitro HOME.
+     # The system user used to run the Vitro web application must have write access
+     # to the parent directory of the directory defined in this property, if Vitro HOME
+     # does not already exist.
+     # If this directory already exists, the system user used to run the Vitro web application
+     # must have write access to this directory.
+   -->
    <Environment
            type="java.lang.String"
            name="vitro/home"
-           value="${vitro-dir}" override="true"/>
+           value="${vivo-dir}" override="true"/>
+
+   <!--
+     # The name of the application (possibly not used).
+   -->
+   <Environment
+           type="java.lang.String"
+           name="vitro/appName"
+           value="vivo" override="true"/>
+
+   <!--
+     # The email address of the root user for the Vitro application. The password
+     # for this user is initially set to "rootPassword", but you will be asked to
+     # change the password the first time you log in.
+   -->
+   <Environment
+           type="java.lang.String"
+           name="vitro/rootUserAddress"
+           value="vivo_root@mydomain.edu" override="true"/>
```

Comment:
I do not think the root user address or the default namespace can be changed after the fact. @brianjlowe, do you by chance know whether they can?

Reply:
We could use java opt variables to allow these to change but do not want to risk breaking on restart.

Reply:
I have added the ability to adjust these via runtime variables with the following fast forward PR for docker updates.

Reply:
This is what happens when changing the root user address on restart.

```
Warning
VIVO issued warnings during startup.

WARNING: RootUserPolicy$Setup
runtime.properties specifies 'vivo_admin@library.tamu.edu' as the value for 'rootUser.emailAddress', but the system contains this root user instead: vivo_root@library.tamu.edu
edu.cornell.mannlib.vitro.webapp.auth.policy.RootUserPolicy$Setup
WARNING: RootUserPolicy$Setup
Creating root user 'vivo_admin@library.tamu.edu'
edu.cornell.mannlib.vitro.webapp.auth.policy.RootUserPolicy$Setup
WARNING: RootUserPolicy$Setup
For security, it is best to delete unneeded root user accounts.
edu.cornell.mannlib.vitro.webapp.auth.policy.RootUserPolicy$Setup
```

Reply:
> 

@wwelling When rootUserAddress and defaultNamespace were defined in vivo_home/config/runtime.properties it was enough to set it up once, and when you upgrade to newer version of VIVO/Vitro you can just point to vivo_home directory.  If I understand correctly, after moving these variables into context.xml when some institution would like to upgrade to newer version of VIVO/Vitro will have to set up again these variables in context.xml. Not sure what should be our general principle for making decision what should be in vivo_home/config, and what should be in context.xml. Do you have any rationality/principle for moving exactly those two variables from vivo_home to context.xml? Is that needed for building or installation process?

Reply:
Here is the original commit moving these properties. https://github.com/vivo-project/Vitro/pull/192/commits/1e936b4fd60071b1faa66fe5ba938117c8d7924a

Reply:
I do not have the exact reason for this to move to JDNI properties.

Reply:
It is defined in this Jira issue however, https://vivo-project.atlassian.net/browse/VIVO-1443.

Reply:
@chenejac lets discuss and make a decision on this as this is blocking remaining installation work already in progress on this sprint.

Reply:
We discussed this, and the conclusion was that those two parameters have to be moved to context.xml, for the properly running of VIVO from the war file. It will enable that someone is not building source code, just downloading and using war in the VIVO upgrade process. In the documentation, we have to clearly state if someone is not building source code, context should be defined in  $CATALINA_BASE/conf/[enginename]/[hostname]/  (usually $CATALINA_BASE/conf/Catalina/localhost/vivo.xml). Also, in documentation we should suggest making digest file before upgrading and manual check of the change log of rdf files in the new version of VIVO.